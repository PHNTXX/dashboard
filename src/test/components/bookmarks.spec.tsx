import { render } from "@testing-library/react";
import BookmarkList, {
  BookmarkGroup,
  IBookmarkGroupProps,
  IBookmarkListProps,
} from "../../components/bookmarks";

const bookmarkGroupProps: IBookmarkGroupProps = {
  name: "Test Group",
  items: [
    {
      name: "Bookmark Test",
      url: "#",
    },
  ],
};

const bookmarkListProps: IBookmarkListProps = {
  groups: [bookmarkGroupProps, bookmarkGroupProps],
};

it("BookmarkGroup snapshot test", () => {
  const { asFragment } = render(
    <BookmarkGroup
      name={bookmarkGroupProps.name}
      items={bookmarkGroupProps.items}
    />,
  );

  expect(asFragment).toMatchSnapshot();
});

it("BookmarkList snapshot test", () => {
  const { asFragment } = render(
    <BookmarkList groups={bookmarkListProps.groups} />,
  );

  expect(asFragment).toMatchSnapshot();
});
