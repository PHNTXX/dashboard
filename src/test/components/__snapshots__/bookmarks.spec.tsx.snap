// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkGroup snapshot test 1`] = `[Function]`;

exports[`BookmarkList snapshot test 1`] = `[Function]`;
